js/components/ReportSummaryForm.vue

```diff
@@ -36,10 +36,10 @@
       </div>
       <div class="row form-group">
         <div class="col strategy-controls">
-          <label for="strategy">Summary Type</label>
+          <label>Summary Type</label>
           <div class="form-check" v-for="(strategyVal, i) in strategies" :key="strategyVal">
-            <input class="form-check-input" :class="{ 'is-invalid': errors.strategy }" :id="'strategy' + i" name="strategy" v-model="model.strategy" :value="strategyVal" type="radio" :disabled="!reportSelected">
-            <label class="form-check-label" :for="'strategy' + i">{{ strategyVal }}</label>
+            <input class="form-check-input" :class="{ 'is-invalid': errors.strategy }" :id="fieldId('strategy' + i)" :name="fieldId('strategy' + i)" data-description="strategy-input" v-model="model.strategy" :value="strategyVal" type="radio" :disabled="!reportSelected">
```

Comment:
While extracting the `fieldId` function, I decided to try it on the `name` attribute and that solved the 3rd issue. I don't fully understand why, and it doesn't seem to be a problem with the other fields.